src/index.ts

```diff
@@ -4,4 +4,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.*
  */
 
-export * from './hue-bridge-emulator'
+import { HueBridge } from './hue-bridge';
+
+let bridge = new HueBridge({ debug: true });
+bridge.start();
```

Comment:
```suggestion
export * from './hue-bridge'
```

It should be still usable as a library.
I'm totally fine if you add something like a `server.js`.